dspace-api/src/test/java/org/dspace/storage/rdbms/migration/V6_0_2016_04_01_0002__DS_2199_Move_IdentifiersIT.java

```diff
@@ -0,0 +1,252 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+
+package org.dspace.storage.rdbms.migration;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import javax.sql.DataSource;
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.dspace.AbstractDSpaceTest;
+import org.dspace.identifier.DOI;
+import org.dspace.identifier.EZIDIdentifierProvider;
+import org.dspace.services.ConfigurationService;
+import org.dspace.services.factory.DSpaceServicesFactory;
+import org.dspace.storage.rdbms.DatabaseDebug;
+import org.dspace.storage.rdbms.DatabaseUtilsHelpers;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Test migration of certain identifiers to new fields, for DS-2199.
+ * This class must do its own database setup, because it tests code that is run
+ * by Flyway before the persistence unit is initialized.
+ *
+ * @author mwood
+ */
+public class V6_0_2016_04_01_0002__DS_2199_Move_IdentifiersIT
+        extends AbstractDSpaceTest
+{
+    private static final String SHOULDER = "10.5072/FK2/";
+
+    private int oldField;
+    private int newField;
+
+    private Connection cnctn;
+
+    public V6_0_2016_04_01_0002__DS_2199_Move_IdentifiersIT()
+    {
+    }
+
+    @BeforeClass
+    public static void setUpClass()
+    {
+    }
+
+    @AfterClass
+    public static void tearDownClass()
+    {
+    }
+
+    @Before
+    public void setUp()
+            throws SQLException
+    {
+        // Set up a DBMS connection
+        DataSource ds = DSpaceServicesFactory.getInstance().getServiceManager()
+                .getServiceByName("dataSource", BasicDataSource.class);
+        cnctn = ds.getConnection();
+
+        // Define the database.
+        DatabaseUtilsHelpers.updateDatabase(ds, cnctn);
```

Comment:
You may be able to do away with the `DatabaseUtilsHelpers` class altogether.  You can simply call `DatabaseUtils.updateDatabase()` (no arguments)